src/Form/CocktailFormType.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use App\Entity\CocktailName;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class CocktailFormType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('name', TextType::class)
+        ;
+    }
+
+    public function configureOptions(OptionsResolver $resolver): void
+    {
+        $resolver->setDefaults([
+            'data_class' => CocktailName::class,
+        ]);
+    }
+
```

Comment:
check again your code for useless spaces

Reply:
fixed